di_website/publications/panels.py

```diff
@@ -0,0 +1,30 @@
+from django import forms
+from wagtail.admin.edit_handlers import FieldPanel
+from .edit_handlers import MultiFieldPanel
+from .edit_handlers import ThemeFieldPanel
+
+
+def ThemePanel(
+    heading='Theme',
+    description='Optional: select a custom theme to override the style of this report and associated child pages.'
+):
+    return MultiFieldPanel(
+        [
+            ThemeFieldPanel('report_theme', classname='non-floated-options', widget=forms.RadioSelect),
+        ],
+        heading=heading,
+        description=description
+    )
```

Comment:
As far as I can tell, this is not being used anywhere ...

Reply:
Removed unused functions